src/main/kotlin/minesweeper/domain/Land.kt

```diff
@@ -0,0 +1,21 @@
+package minesweeper.domain
+
+class Land(
+    mines: Mines,
+    val point: Point,
+    private var isOpened: Boolean = false,
+) {
+    val aroundMineCount: Int
+
+    init {
+        aroundMineCount = countAroundMines(mines)
+    }
+
+    fun isOpened(): Boolean = isOpened
+
+    fun open() {
+        isOpened = true
+    }
+
+    private fun countAroundMines(mines: Mines): Int = Direction.applyTo(point).count { point -> Mine(point) in mines }
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

코틀린 공식 문서에 따르면 함수 체인의 경우 아래와 같이 체인별 개행을 해주어야 합니다.
(일반적으로 체인의 첫 번째 호출에는 개행이 있어야 하지만 개행하지 않는 편이 더 합리적이라면 생략해도 괜찮습니다)

```suggestion
    private fun countAroundMines(mines: Mines): Int = Direction
        .applyTo(point)
        .count { point -> Mine(point) in mines }
```

프로젝트 전반적으로 적용해주시면 좋을 것 같아요!

Reply:
`countAroundMines`가 수행하고 있는 로직도 테스트 시나리오에서 검증해보면 어떨까요?

---

가독성의 이유만으로 분리한 private 함수의 경우 public으로도 검증 가능하다고 여겨질 수 있습니다. public 함수가 private 함수를 사용하고 있기 때문에 자연스럽게 테스트 범위에 포함될 수 있습니다.
하지만 가독성 이상의 역할을 하는 경우 testable하게 구현하기 위해서는 해당 역할을 수행하는 다른 객체를 만들거나, 메세지를 설계해볼 타이밍이 아닐지 고민해볼 수 있습니다.

Reply:
음.. 그렇군요! 저는 별 이유는 없었고.. 짧을땐 굳이 개행을 안하면 더 읽기 쉽다고 생각했습니다!
다만 코틀린 공식 문서에서 체인별 개행을 암묵적인 컨벤션으로 사용하고 있는걸 보니 다른 코틀린 사용자에게도
익숙한 형태로 보이게 하는 것이 더 좋겠네요. 피드백 감사합니다 👍

Reply:
> countAroundMines가 수행하고 있는 로직도 테스트 시나리오에서 검증해보면 어떨까요?

음 .. 생성 테스트 결과로 검증이 가능하다고 생각했어서 따로 테스트를 작성할 생각을 못했었네요.
근데 지뢰찾기라는 도메인에서 특정 땅 주변 지뢰  개수는 중요한 비즈니스 로직이므로 말씀처럼 testable한 객체가 필요할 것 같습니다.

객체로 추출해보겠습니다!